enterprise/frontend/src/embedding-sdk/hooks/private/setup-sdk-auth.unit.spec.tsx

```diff
@@ -1,63 +0,0 @@
-import type { SDKConfig } from "embedding-sdk";
```

Comment:
Everything in here should be tested by higher level tests, these tests were highly coupled with the implementation